app/src/main/java/org/mozilla/focus/home/pocket/Pocket.kt

```diff
@@ -79,7 +81,8 @@ object Pocket {
      */
     @AnyThread // videosCache is synchronized.
     fun getRecommendedVideos() = videosCache
-    private fun getRecommendedVideosNoCacheAsync() = async { PocketEndpoint.getRecommendedVideos() }
+    private fun getRecommendedVideosNoCacheAsync(placeholderFlag: Boolean) =
```

Comment:
Personal opinion, but I usually think that there are more readable alternatives to passing flags to change method behavior.  If we broke this into two methods I think we would get a few benefits.

- At the top level (inside the init block) it would be clear what was going on.  `getRecommendedVideosNoCacheAsync(true)` can be ambiguous.  `getRecommendedVideosNoCacheAsync` vs `getPlaceholderVideos` is immediately clear
- We wouldn't have to pass this flag through multiple levels of code, which makes maintenance simpler.
- We could pull the conditional statement out of PocketEndpoint.  We would then be less likely to confuse developer ergonomics code with prod code when debugging.